cfgov/unprocessed/apps/owning-a-home/css/form-explainer.less

```diff
@@ -102,184 +96,94 @@
         }
     }
 
-    .o-expandable-group .o-expandable__form-explainer {
-        display: none;
-        margin: 0 0 unit( 3px / @base-font-size-px, em );
-        border-bottom: none;
+    // Expandable overrides
+
+    .o-expandable-group .o-expandable {
+        border: 4px solid transparent;
+        box-sizing: border-box;
         transition: transform 100ms ease-out;
-        transform: translate3d( 0, 0, 0 );
-        will-change: transform;
+        margin-bottom: 5px;
 
-        &.o-expandable__expanded,
-        &.o-expandable__expanding,
-        &.has-attention {
-            transform: translate3d( -( @grid_gutter-width / 2 ), 0, 0 ) !important;
-            .o-expandable_link {
-                color: @black;
-            }
+        &:first-child {
+            border-top: 4px solid transparent;
         }
 
-        &:hover {
+        &.has-attention {
+            transform: translateX( -15px );
+
             .o-expandable_link {
                 color: @black;
             }
         }
 
-        .o-expandable_header {
-            z-index: 1000;
-        }
-
-        .o-expandable_header-left {
-            max-width: 90%;
-        }
-
-        &:last-child .o-expandable_content {
-            border-bottom: none;
-        }
-
-        .o-expandable_header {
-            padding: unit( ( @grid_gutter-width/2 )/ @base-font-size-px, em )
-                     unit( @grid_gutter-width / @base-font-size-px, em )
-                     unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+        .o-expandable_content__expanded:before {
             border: none;
         }
 
         .o-expandable_label {
-            font-size: unit( @base-font-size-px / 16 * 100, % );
-            line-height: @base-line-height;
+            font-size: 16px;
```

Comment:
Should this be a calculated `unit()` size?

Reply:
Fixing.